src/render.rs

```diff
@@ -61,6 +62,7 @@ struct Config {
     height_in_tiles: u32,
     target_width: u32,
     target_height: u32,
+    clear_color: u32,
```

Comment:
`struct Config` (Rust) and `struct Config` (wgsl) need comments specifying that they must be kept in sync in the correct order.

This is as good a time as any to add them

Reply:
Good point! Done.